SPECS/linux-firmware/linux-firmware.spec

```diff
@@ -17,6 +17,30 @@ BuildArch:      noarch
 %description
```

Comment:
Question - Today if a user installs `linux-firmware` RPM, they'll get all the firmware blobs. Is it possible to preserve this behavior now that we're splitting the firmwares out into sub-packages? (i.e., `linux-firmware` package itself has `Requires: linux-firmware-XXXXX` for each sub-package?)

Doing something like this may help prevent a breaking change since the user was originally expecting all the firmware blobs to be installed with `sudo dnf install linux-firmware`.

Reply:
I tried this approach adding all sub package as Requires but toolkit was throwing error while building graph for linux-firmware package.

Reply:
Can we bottom out on the graph error? Thinking through the scenario more, I think we should find a way to preserve the current behavior where updated `linux-firmware` package still distributes the firmware blobs that were present before.

Consider the situation where this updated `linux-firmware` package with minimal firmware blobs is published, a functioning production system that performs a package update will lose its firmware blobs since the older `linux-firmware` package will be uninstalled on the update operation. One of those firmware blobs could be required for the system's networking or storage driver to function.

Reply:
Made required change and looks like it is working and no repro of unresolved dependency error. We need Requires: linux-firmware-XXXXX = %{version}-%{release}